circle.yml

```diff
@@ -122,7 +122,7 @@ jobs:
           name: Install build dependencies
           command: |
             apt-get -qq update
-            apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libz3-dev
+            apt-get -qy install cmake libboost-regex-dev libboost-filesystem-dev libboost-test-dev libboost-system-dev libboost-program-options-dev libz3-dev libjsoncpp-dev
```

Comment:
Nice! Can you programatically ensure that this install jsoncpp <1.8.0?

Reply:
updated pull request. let me know if this is what in ur mind